Revert/Config.java

```diff
@@ -0,0 +1,108 @@
+import java.nio.charset.Charset;
+//import java.nio.file.Files;

```

Comment:
Rozumiem, że to wersja robocza i stąd komentarze, jednak w praktyce każdy zakomentowany kod to zło ;)